app/static/js/eventKiosk.js

```diff
@@ -61,18 +97,22 @@ function submitData(hitEnter = false){
         "eventid": $("#eventid").val(),
         "bNumber": $("#submitScannerData").val()
       },
-      success: function(result) {
+      success: function(decodedText) {
```

Comment:
what is passed into this function is not decoded text. If you are going to rename it from result, name it something more meaningful.